fuzztest/internal/subprocess.cc

```diff
@@ -32,6 +32,17 @@
 #include <string>
 #include <vector>
 
+#if defined(__APPLE__)
```

Comment:
This is getting quite convoluted compared to the other OS checks, so you could do something like this instead:

```c++
#if defined(__APPLE__)
#include <TargetConditionals.h>
#endif

// ...

#if !defined(_MSC_VER) && \
    ... && \
    !(defined(TARGET_OS_TV) && TARGET_OS_TV)
#endif
```